solution/backend/regulations/admin.py

```diff
@@ -98,11 +120,17 @@ def import_conversions(self, text, url, act):
             }
             if section in toc:
                 data["name"] = toc[section]["name"]
-                data["statute_title"] = toc[section]["statute_title"]
+                title = toc[section]["statute_title"]
+                try:
+                    title = int(title)
+                except ValueError:
+                    title = roman_to_int(title)
+                if title:
```

Comment:
is if title necessary.  No matter what title should be assigned due to exception catching .